server/sonar-server/src/main/java/org/sonar/server/computation/task/projectanalysis/measure/Measure.java

```diff
@@ -133,11 +151,20 @@ public NewMeasureBuilder setQualityGateStatus(QualityGateStatus qualityGateStatu
       return this;
     }
 
+    /**
+     * @deprecated as only one period is now available. Use {@link #hasVariation()} instead
+     */
+    @Deprecated
     public NewMeasureBuilder setVariations(MeasureVariations variations) {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)